Upgrade ledgerblue dependency

chrisCryptoCom opened the pull request:
Ledgerblue 0.1.42 has an issue that leads to failing installs with newer setuptools versions.
See: https://github.com/LedgerHQ/blue-loader-python/issues/104

This PR bumps the dependency and thereby resolves https://github.com/bandprotocol/pyband/issues/29

RogerKSI commented:
Hi, @chrisCryptoCom, thanks for your contribution. 

There's an issue with the poetry lock file in the test. I just updated the version.
Could you sync the branch and regenerate the lock file with poetry 1.4.1?

chrisCryptoCom commented:
> Hi, @chrisCryptoCom, thanks for your contribution.
> 
> There's an issue with the poetry lock file in the test. I just updated the version. Could you sync the branch and regenerate the lock file with poetry 1.4.1?

Sure thanks for the hint, this is the first time for me working with poetry😅
I merged in the master branch, deleted the lockfile and regenerated it with `poetry install`. Hope this fixes the issue.

taobun reviewed (approved):
LGTM. We will fix our test workflow after merge this PR.
